doc/api/next_api_changes/2018-07-22-AL-deprecations.rst

```diff
@@ -5,3 +5,6 @@ Support for custom backends that do not provide a ``set_hatch_color`` method is
 deprecated.  We suggest that custom backends let their ``GraphicsContext``
 class inherit from `GraphicsContextBase`, to at least provide stubs for all
 required methods.
+
+The fields ``Artist.aname`` and ``Axes.aname`` are deprecated. Please use
+``isinstance()`` or ``__class__.__name__`` checks instead.
```

Comment:
Well *technically* that's not the same (all subclasses of Axes, e.g. AxesSubplot, have "Axes" as aname, and similarly for Artist), but I agree that's not even close to being a reason not to deprecate it...